app/controllers/responsible_body/devices/chromebook_information_controller.rb

```diff
@@ -16,7 +16,8 @@ def update
       { school: @school }.merge(chromebook_params),
     )
     if @chromebook_information_form.valid?
-      @preorder_info.update!(chromebook_params.merge(status: @preorder_info.infer_status))
+      @preorder_info.update!(chromebook_params)
```

Comment:
Could this be pushed into a method on the PreorderInformation class e.g. `@preorder_info.update_chromebook_information!(chromebook_params)` which then handles the changes and updates the status?

Reply:
Good call - done